sale.example.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
The format is now single file with a list of sale configs

Reply:
Will update to match